arches_her/views/map.py

```diff
@@ -0,0 +1,41 @@
+from django.views.generic import View
+from django.db import connection
+from django.http import HttpResponse, Http404
+from arches.app.models import models
+from arches.app.utils.permission_backend import get_restricted_instances
+
+
+class ApplicationAreas(View):
+    def get(self, request, zoom, x, y):
+        nodeid = '1909956f-3a3b-11eb-ae99-f875a44e0e11'
+        if hasattr(request.user, "userprofile") is not True:
+            models.UserProfile.objects.create(user=request.user)
+        viewable_nodegroups = request.user.userprofile.viewable_nodegroups
+        
+        try:
+            node = models.Node.objects.get(nodeid=nodeid, nodegroup_id__in=viewable_nodegroups)
+            resource_ids = get_restricted_instances(request.user, allresources=True)
```

Comment:
You probably don't want to use 'allresources' here. That will filter out any resource that any user does not have access to. It's really a problem with the api.py view that was put into place to support caching because of performance issues with that view. However, this view is much faster and caching isn't needed, so you can filter out just those instances that the current user is not allowed to see. You can see how I did this in my PR.

Reply:
@chiatt Thanks for the explanation - that really helps.  I'll make the changes as needed.

Reply:
suggested change has been applied